# Reorder include directories to fix build when ffmpeg-0.7 is installed
#
--- ./services/fileindexer/indexer/CMakeLists.txt.orig	2013-03-07 02:19:03.114665070 +0000
+++ ./services/fileindexer/indexer/CMakeLists.txt	2013-03-07 02:19:11.568663110 +0000
@@ -108,7 +108,7 @@
 endif(EXIV2_FOUND)
 
 if(FFMPEG_FOUND)
-    include_directories( ${FFMPEG_INCLUDE_DIRS} )
+    include_directories(BEFORE ${FFMPEG_INCLUDE_DIRS} )
 
     kde4_add_plugin( nepomukffmpegextractor ffmpegextractor.cpp extractorplugin.cpp )
 
